pyqtgraph/widgets/ComboBox.py

```diff
@@ -230,7 +229,7 @@ def saveState(self):
             except AttributeError:
                 pass
         if data is None:
-            return asUnicode(self.itemText(ind))
+            return str(self.itemText(ind))
```

Comment:
QComboBox::itemText(index) returns a string as well, so we can remove the casting here too